pylint/checkers/classes/class_checker.py

```diff
@@ -1454,6 +1459,24 @@ def _check_invalid_overridden_method(
                 node=function_node,
             )
 
+    def _check_functools_or_not(self, decorator: nodes.Attribute) -> bool:
+        if decorator.attrname != "cached_property":
+            return False
```

Comment:
What if we have some import that is ``from functools import cached_property as cached`` ? But let's hear from someone else first, I'm not sure handling alias perfectly is that important, aliasing doesn't happen often and it's bad practice imo.

Reply:
Yeah, I let's leave that out of scope. We didn't even do that for `TYPE_CHECKING` even though we did some elaborate work for aliasing `typing`.

Reply:
Not sure about it but we could revert 8d62ba0a0b55f24d010082b8e8f484ab7d816f37 then ?

Reply:
Nah, I think we need it because it makes the aliasing functools as ft test case work.